perf-tool/src/client.cpp

```diff
@@ -147,7 +171,8 @@ void Client::receiveMessage(char buffer[])
 
     if (n > 0)
     {
-        printf("Received: %s\n", buffer);
+        fputs(buffer, clientLogFile);
+        fputs(",\n", clientLogFile);
```

Comment:
Just realised adding newline here is not the correct place as we get this data in chunks.

Reply:
Right